2.2.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <meta charset="UTF-8" ; />
+        <body>
+                <p><span id="demo"></span>.</p>
+            <script>
+                var num = [1,2,3,4,5,6];
+                document.write(num+" ");
+                num.reverse();
```

Comment:
I like it :D
The first task could be done like that too, but please try to implement your own reverse function.

Reply:
Done